Documentation/DevelopmentTips.md

```diff
@@ -178,6 +178,20 @@ For example:
 	$ tools/scripts/xabuild /t:DeployTestApks tests/locales/Xamarin.Android.Locale-Tests/Xamarin.Android.Locale-Tests.csproj
 	$ tools/scripts/xabuild /t:RunTestApks    tests/locales/Xamarin.Android.Locale-Tests/Xamarin.Android.Locale-Tests.csproj
 
+## Running `.apk` Projects with Include/Exclude
+
+For example, to exclude tests that use the internet (`InetAccess` category):
+
+	$ make run-apk-tests EXCLUDECATEGORIES=InetAccess
+
+On Windows:
+
+	$ msbuild Xamarin.Android.sln /t:RunApkTests /p:ExcludeCategories=InetAccess
+
+`INCLUDECATEGORIES` and `IncludeCategories` function in the same fashion.
+
+To specify multiple categories, delimit each category with a `,` character.
```

Comment:
*Should* `,` be used as a delimiter?

The problem is, it *can't* be used on the command-line:

```
$ msbuild /nologo Xamarin.Android.sln /t:RunApkTests /p:ExcludeCategories=InetAccess,Bologna
MSBUILD : error MSB1006: Property is not valid.
Switch: Bologna

For switch syntax, type "MSBuild /help"
```

This is why `:` was chosen as the separator char for e.g. `$(AndroidSupportedTargetJitAbis)`, as it *can* be specified on the command-line.